### PR TITLE
DOC: remove "python -m" and "--symbols" from the example

### PR DESCRIPTION
**Pull request by wilsonkichoi**

### CODE REVIEW

**Comment by llllllllll** [comment]
Thanks! Can we add a line about running `zipline ingest` first?

**Comment by wilsonkichoi** [comment]
ingest command added

File: docs/source/beginner-tutorial.rst [hunk 1]
```diff
@@ -96,6 +96,20 @@ To now test this algorithm on financial data, ``zipline`` provides three
 interfaces: A command-line interface, ``IPython Notebook`` magic, and
 :func:`~zipline.run_algorithm`.
 
+Ingesting Data
+^^^^^^^^^^^^^^
+If you haven't ingest the data, run:
```
Comment (richafrank) [inline docs/source/beginner-tutorial.rst]: Sorry to slow this down - could you just fix the typo: "ingest" should be "ingested", and we'll merge this.  Really appreciate the doc improvements here.

File: docs/source/beginner-tutorial.rst [hunk 2]
```diff
@@ -96,6 +96,20 @@ To now test this algorithm on financial data, ``zipline`` provides three
...
+
+.. code-block:: bash
+
+   $ zipline ingest [-b <bundle>]
+
+where ``<bundle>`` is the name of the bundle to ingest, defaulting to
+:ref:`quantopian-quandl <quantopian-quandl-mirror>`.
+
+you can check out the :ref:`ingesting data <ingesting-data>` section for
+more detail
```
Comment (richafrank) [inline docs/source/beginner-tutorial.rst]: period.

**Pull request merged**